.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+.pyc
```

Comment:
lib/gag.pyc is this present in git. you'll have to use pattern here

Reply:
Updated with the pattern